terraform/application/config/qa.tfvars.json

```diff
@@ -1,6 +1,6 @@
 {
     "cluster": "test",
     "namespace": "bat-qa",
-    "deploy_azure_backing_services": false,
-    "enable_postgres_ssl": false
+    "enable_postgres_ssl": true, 
+    "inf_vault_name": "s189t01-faltrn-dv-inf-kv"
   }
```

Comment:
Missing final new line. It can be added automatically when the vscode option is selected: https://dfedigital.atlassian.net/wiki/spaces/BaT/pages/1905491978/Onboarding#VSCode-configuration